src/Electrical/Analog/ideal_components.jl

```diff
@@ -80,27 +65,20 @@ function Capacitor(;name, C=1.0, v_start=0.0)
 end
 
 """
-```julia
-function Inductor(; name, L = 1.0)
-```
+    Inductor(; name, L = 1.0)
 
 Creates an ideal Inductor.
 
 # States
-- `v(t)`: [`V`]
-  The voltage across the inductor, given by `D(p.i) ~ v / L`
+See [OnePort](@ref)
 
 # Connectors
-- `p`
-  Positive pin
-- `n`
-  Negative pin
+- `p` Positive pin
+- `n` Negative pin
 
 # Parameters:
-- `L`: [`H`]
-  Inductance
-- `i_start`: [`A`]
-  Initial current through inductor
+- `L`: [`H`] Inductance
+- `i_start`: [`A`] Initial current through inductor
 """
 function Inductor(;name, L=1.0e-6, i_start=0.0)
```

Comment:
```suggestion
function Inductor(;name, L, i_start=0.0)
```
unrelated to this PR, but default value seems quite arbitrary.

Reply:
Agreed, anything like this we should require.